### PR TITLE
CVT-9950 - Set up Github actions to run PHPUnit, PHPStan and PHPCSFixer

### PR DESCRIPTION
**Pull request by yevhen-minenko**

### CODE REVIEW

File: .github/workflows/on-pull-request.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+on:
+  pull_request:
```
Comment (avfletch) [inline .github/workflows/on-pull-request.yml:2]: CONSIDER: Can we make this run on merge to main or prevent that?

**Review by avfletch — CHANGES_REQUESTED** [review]
Such review.  Very comment.

File: .github/workflows/on-pull-request.yml (hunk 1 above)
Comment (yevhen-minenko) [reply .github/workflows/on-pull-request.yml:2]: It's just a reply

**Review by avfletch — APPROVED** [review]
This looks good to me.

**Pull request merged**